jdbc-adapter/src/main/java/org/polypheny/db/adapter/jdbc/JdbcSchema.java

```diff
@@ -167,7 +167,7 @@ public JdbcTable createJdbcTable( CatalogTable catalogTable, List<CatalogColumnP
                 physicalSchemaName,
                 partitionPlacement.physicalTableName,
                 physicalColumnNames );
-        tableMap.put( catalogTable.name, table );
+        tableMap.put( catalogTable.name + "_" + partitionPlacement.partitionId, table );
```

Comment:
@vogti This was necessary because for every table modification the table will be looked up via the current Schema (other) in the specific case of `RelToSqlConverter`. However here the table name is only the logical table name.

Which results in an exception because it can't find a tablemapping from table_partitionId in the tableMap.

If this is okay for you I would change this in all necessary files
![image](https://user-images.githubusercontent.com/64148462/129524407-10aa9943-2b34-4175-aaae-746082b116e6.png)

Reply:
Yes, this makes sense.